packages/odyssey-icons/src/plus.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24"><path fill="currentColor" d="M13 3C13 2.44772 12.5523 2 12 2C11.4477 2 11 2.44772 11 3V11H3C2.44772 11 2 11.4477 2 12C2 12.5523 2.44772 13 3 13H11V21C11 21.5523 11.4477 22 12 22C12.5523 22 13 21.5523 13 21V13H21C21.5523 13 22 12.5523 22 12C22 11.4477 21.5523 11 21 11H13V3Z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon"><path fill-rule="evenodd" clip-rule="evenodd" d="M6 12C6 12.5523 6.44772 13 7 13C7.55228 13 8 12.5523 8 12V8L12 8C12.5523 8 13 7.55228 13 7C13 6.44772 12.5523 6 12 6L8 6V2C8 1.44772 7.55228 1 7 1C6.44772 1 6 1.44772 6 2V6L2 6C1.44772 6 1 6.44771 1 7C1 7.55228 1.44772 8 2 8L6 8V12Z" fill="currentColor"/></svg>
```

Comment:
why add the class="ods-icon" code here if we're going to inject it with SVGO post processing?

Reply:
The naming on these directories is wonky for now. @jmelberg-okta and I went back and forth a bit trying to figure out what works for now.

Today, our SVGO optimization is done manually - not during compilation - because we are visually checking every icon.  *For now*, `figma` is what would normally constitute `src`, and `src` is what would normally constitute `dist`. This is mainly an indicator that there is no compilation step yet.

Once we add visual regression testing, we'll automate optimization, drop Figma exports into `src` and treat `dist` as the deliverable as normal.